.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
         run: docker build -t test-cargo-deny .
 
       - name: Run list
-        run: docker run -v ${PWD}/test:/test test-cargo-deny --context test list
+        run: docker run -v ${PWD}/test:/test test-cargo-deny --manifest-path test/Cargo.toml list
 
       - name: Run check
-        run: docker run -v ${PWD}/test:/test test-cargo-deny --context test check
+        run: docker run -v ${PWD}/test:/test test-cargo-deny --manifest-path test/Cargo.toml check
```

Comment:
Looking at our changelog 0.7.0 contained this change as well which will change the behavior of the action as it currently is to only include default features, instead of all features.

> Resolved #216 by adding support for the --all-features, --features, and --no-default-features flags to specify the exact features to have enabled when gathering the crates in your dependency graph to actually run checks against. This is a BREAKING CHANGE as previously crates were gathered with --all-features.

Not fully sure what is best in the action here as default, I am a bit wary of changing the default as repos using it may be reliant on the current default to include all features. So we could for now include `--all-features` in the action here still to keep the same default behavior for it. And then later add option to the action to change the default behavior (or do it in the next v2 version of it). Thoughts @Jake-Shadle ?

This is the type of thing we have to be very aware and careful about with cargo-deny vs the action.

Reply:
I'm kind of leaning towards having an `args` input for the action, that is used if set, otherwise `--all-features` and the current log-level/command inputs are used to generate the command, as otherwise we need to add inputs for all the different options, as one of the nice things with --all-features is that it works at both an individual crate level and workspace level, so if people want to run cargo-deny on a specific crate with specific features, the action would already get in their way at the moment.

Reply:
Very good point about keeping the same behavior as the 06-version of the action. I should probably have read the changelog better :-)

I've added a new optional input `arguments` that by default has the value `--all-features`, following @Jake-Shadle´s suggestion, and updated the README.

WARNING and CAVEAT: I tried to test the changes I made by [pointing out my repository as the cargo-deny-action in a minor repo](https://github.com/zayenz/minizinc-pentominoes-generator/blob/main/.github/workflows/deny.yml), but unfortunately it seems [to pull an old version of the action](https://github.com/zayenz/minizinc-pentominoes-generator/runs/1030039394?check_suite_focus=true) so I haven't been able to test it yet. It might just be some minor thing, but I haven't worked with GitHub actions a lot before, so I'm kind of new to the area.

Reply:
After posting the above, I realized that the tag needed to be updated. In my fork I added a `v1-beta` tag, and then I updated my test-repository, which now worked as intended: https://github.com/zayenz/minizinc-pentominoes-generator/runs/1030095790?check_suite_focus=true